includes/ajax-handlers/class-wc-stripe-checkout-sessions-ajax-handler.php

```
@@ -15,6 +15,41 @@ class WC_Stripe_Checkout_Sessions_Ajax_Handler {
+		try {
+			$is_nonce_valid = check_ajax_referer( 'wc_stripe_get_checkout_session_status_nonce', 'security', false );
+			if ( ! $is_nonce_valid ) {
+				throw new Exception( __( "We're not able to process this request. Please refresh the page and try again.", 'woocommerce-gateway-stripe' ) );
+			}
+
+			$session_id = isset( $_POST['session_id'] ) ? sanitize_text_field( wp_unslash( $_POST['session_id'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Missing
+			if ( empty( $session_id ) ) {
+				throw new Exception( __( 'Missing checkout session ID.', 'woocommerce-gateway-stripe' ) );
+			}
+
+			$checkout_session = WC_Stripe_API::request( [], "checkout/sessions/{$session_id}", 'GET' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_API' in method 'get_checkout_session_status'. (undefined)

(StaticAccess)